app/controllers/api/avatars.php

```diff
@@ -154,11 +156,42 @@
             'id' => $githubId
         ];
     } catch (Exception $error) {
+        if ($logger) {
+            $version = System::getEnv('_APP_VERSION', 'UNKNOWN');
+
+            $log = new Log();
```

Comment:
why's this code here? We have global handlers for error right?

Reply:
This is an old code implantation. Maybe because it's being called inside a controller and we don't want the effect to propagate to the end user